pareto/tests/test_strategic_model.py

```diff
@@ -638,7 +644,7 @@ def test_run_reduced_strategic_model(build_reduced_strategic_model):
     assert results.solver.status == pyo.SolverStatus.ok
     assert degrees_of_freedom(m) == 11789
     # solutions obtained from running the reduced generic case study
-    assert pytest.approx(89201.666, abs=1e-1) == pyo.value(m.v_Z)
+    assert pytest.approx(88199.598, abs=1e-1) == pyo.value(m.v_Z)
```

Comment:
@nareshsusarla this is still showing a slight change in the objective function value - do you have an explanation?

Reply:
Note: change in objective is expected as previous versions of small_strategic_model contain no values for PipelineOperationalCosts, meaning the default value is being used (0.01 instead of 0.0001 $/bbl)